### ISSUE TITLE
Single circular ROI extraction: automatically switch to `extract_single_roi` function

### ISSUE THREAD

**Issue by emarin2642** [Codra-Ingenierie-Informatique/DataLab#31]
**Describe the bug**
ROI extraction is different than the definition.

In the first image below the circle ROI is defined very close to the central ring. When extraction, see the second image, we can still a little portion of the second ring what is ununexpected !

![ROI_extraction](https://github.com/Codra-Ingenierie-Informatique/DataLab/assets/156417789/eb4e3121-532c-41ad-ad0f-ffdb51418c43)
![ROI_extraction_2](https://github.com/Codra-Ingenierie-Informatique/DataLab/assets/156417789/28be586c-a803-4e89-ada7-58dd12f623a9)

**Comment by PierreRaybaut** [comment]
Thanks for the feedback.

I was almost tempted to answer that it's a feature, not a bug 😉, but at least this is the intended behavior for now even if it may be seen as inappropriate.

The current behavior of the **circular** ROI is to be able to do the computations (all features in the "Computation" menu) only in the area of interest (inside the circle). However, when extracting the ROI, we extract the smallest square area around the circle.

Actually, there might be an issue here because at some point we used to keep the circular ROI in the extracted image, but we removed this feature: perhaps we should reimplement it.

An alternative solution could be to replace the pixel values outside the ROI area by a certain value (which would be asked to the user: 0, by default for example).
